docs/src/main/asciidoc/logging.adoc

```diff
@@ -56,24 +51,22 @@ public class ExampleResource {
 ----
 
 NOTE: While JBoss Logging routes log messages into JBoss Log Manager directly, one of your libraries might rely on a different logging API.
-In such cases, you need to use a <<logging-adapters,logging adapter>> to ensure that its log messages are routed to JBoss Log Manager as well.
-
+In such cases, you need to use a <<logging-apis, logging adapters>> to ensure that its log messages are routed to JBoss Log Manager as well.
```

Comment:
The original was correct. In this sentence you have the phrase "you need to use a logging adapters" which doesn't work.